Allow usage of rx.disposeBag alongside rx_disposeBag

freak4pc opened the pull request:
It seems to me having disposeBag scoped into the `rx.` prefix feel more in-line with the rest of How RxSwift 3.0 behaves, so felt this would be a nice touch-up. Of course totally open to any feedback. 

This allows using `.addDisposableTo(rx.disposeBag)`

Thanks! :-)

freak4pc committed:
Allow usage of rx.disposeBag alongside rx_disposeBag

It seems to me having disposeBag scoped into the `rx.` prefix feel more in-line with the rest of the library's behavior, so felt this would be a nice touch-up. Of course totally open to any feedback. 

This allows `.addDisposableTo(rx.disposeBag)`

freak4pc commented:
Seems like the integration test is failing for every PR pushed ? 

Error seems to indicate perhaps some clean needs to be made, or add the specific RxSwift version to the Podfile ? I saw another suggestion of running `pod repo update` and commit those changes to the CI, but that would probably be a different PR ...

ashfurrow commented:
Yeah, CI is currently broken (tracking in #23).

This is a clever workaround, would you mind adding a note in the changelog first? Thanks!

freak4pc committed:
Updated Changelog for PR #29

Updated Changelog.md for PR #29 - Allow usage of rx.disposeBag alongside rx_disposeBag

freak4pc commented:
Sure thing, Done :) Hope this looks fine ! 

Thanks for the feedback 💯

ashfurrow commented:
Thanks a lot for contributing! I've invited you to join the RxSwiftCommunity GitHub organization – no pressure to accept! If you'd like more information on what that means, check out our [contributor guidelines](https://github.com/RxSwiftCommunity/contributors) and feel free to reach out with any questions.

freak4pc commented:
Thanks a lot @ashfurrow . Gladly accept and hope to keep on contributing in any ways possible :)

One more thing I noticed about this is I didn't include a setter for the `rx.disposeBag`, so there's no way to currently manually dispose it by `self.rx.disposeBag = DisposeBag()`. 

Should that be pushed directly or as a separate PR ?

ashfurrow commented:
Let's put it in a separate PR, no rush!
